cosmo_tester/test_suites/general/aio_replace_certs_test.py

```diff
@@ -17,6 +17,10 @@ def test_aio_replace_certs(image_based_manager, ssh_key, logger, test_config):
                                       replace_certs_config_path,
                                       ssh_key.private_key_path)
 
+    local_new_ca_path = join(str(module_tmpdir), 'new_ca.crt')
+    image_based_manager.get_remote_file('~/.cloudify-test-ca/ca.crt', local_new_ca_path)
+    image_based_manager.client._client.cert = local_new_ca_path
```

Comment:
This would be better using the image_based_manager.download_rest_ca(force=True) and image_based_manager.client = image_based_manager.get_rest_client() after the certs replace operation runs.

Reply:
Whoops, no need for me to comment that, it happened in the next commit.